frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/components/editBasicInfo.jspf

```diff
@@ -22,7 +22,7 @@
         <td>
             <div class="form-group">
                 <label class="mandatory" for="comp_name">Name</label>
-                <input id="comp_name" name="<portlet:namespace/><%=Component._Fields.NAME%>" type="text" placeholder="Enter Name" required class="form-control"
+                <input id="comp_name" name="<portlet:namespace/><%=Component._Fields.NAME%>" type="text" placeholder="Enter Name" required pattern=".*\S.*" class="form-control"
```

Comment:
If you want to avoid trailing white-space the correct patter would be `pattern="\S.*\S"` (but this also enforces a length of at least 2 chars

Reply:
Actually this pattern is used to solve problem where user can add only white space characters for ComponentName like `           `.
` pattern=".*\S.*" ` will prevent user from doing that from UI . However user can add something like 
`     Component123   `  with leading and trailing white-spaces but in the backend it will trim the whitespaces and save in DB.

Reply:
@maxhbr would only whitespace search be OK here?

Reply:
Its OK